apps/worker/src/transaction_notifier/notifier.module.ts

```diff
@@ -68,7 +69,7 @@ import { BlockchainService } from '../../../../libs/common/src/blockchain/blockc
       },
     ),
   ],
-  providers: [TxnNotifierService, BlockchainService, ConfigService],
-  exports: [BullModule, TxnNotifierService, BlockchainService, ConfigService],
+  providers: [BullModule, TxnNotifierService, BlockchainService, EnqueueService, ConfigService],
```

Comment:
Out of curiosity, why does BullModule need to be in `providers`?

Reply:
I think I was running into something and put it there then forgot to delete it. Can remove.